core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -425,8 +434,25 @@ private void tryStart(Instant startedAt) {
             // For all registered output consumers, start following as close to container startup as possible
             this.logConsumers.forEach(this::followOutput);
 
+            // Wait until inspect container returns the mapped ports
+            Callable<Integer> inspectMappedExposedPortsCount = () -> {
+                containerInfo = dockerClient.inspectContainerCmd(containerId).exec();
```

Comment:
Can't we use `containerInfo = await()...`?

Reply:
Yes, this was the question in the Slack, which variant to prefer. With this comment answering the question 😉